x2gbfs/providers/cambio.py

```diff
@@ -0,0 +1,153 @@
+import json
+import logging
+from typing import Dict, Optional, Tuple
+
+from x2gbfs.gbfs.base_provider import BaseProvider
+from x2gbfs.util import get
+
+logger = logging.getLogger(__name__)
+
+
+class CambioProvider(BaseProvider):
+    """
+    The CambioProvider retrieves (static) stations and vehicleClasses from Cambio API.
+
+    """
+
+    DEFAULT_MAX_RANGE_ELECTRIC = 200000
+    DEFAULT_MAX_RANGE_COMBUSTION = 600000
+    STATIONS_URL = 'https://cwapi.cambio-carsharing.com/opendata/v1/mandator/{city_id}/stations'
+    VEHICLE_TYPES_URL = 'https://cwapi.cambio-carsharing.com/opendata/v1/mandator/{city_id}/vehicles'
+
+    def __init__(self, feed_config):
+        self.city_id = feed_config['provider-info']['city_id']
+        self.config = feed_config
+
+    def _all_stations(self) -> dict:
+        response = get(self.STATIONS_URL.replace('{city_id}', self.city_id))
```

Comment:
You can use Pythons internal string replacement / formatting mechanism: `self.STATIONS_URL.format(city_id=self.city_id)` (at vehicle types, too)

Reply:
Implemented with 3199d3d795b1fe1ee39ccfa456636495385aa9a3